modules/distribution/resources/operation_policies/specifications/addQueryParams.json

```diff
@@ -1,7 +1,7 @@
 {
   "category": "Mediation",
-  "name": "addQueryParams",
-  "displayName": "Add a query param",
+  "name": "addQueryParam",
```

Comment:
@ashera96 If we change the spec name, we need to update the definition name as well. also this json file's name.